specification/security/resource-manager/readme.md

```diff
@@ -68,8 +68,23 @@ directive:
     where: $.paths
     from: applications.json
     reason: Suppression of OperationsAPI as it doesn't apply to this specific file.
+  suppress: TopLevelResourcesListBySubscription
```

Comment:
Syntax error, it should be "- suppress"

Reply:
After this is fixed, I guess there would be new Lint errors, please also check and fix them.